packages/commons/src/repositories/ReadModelRepository.ts

```diff
@@ -37,7 +37,7 @@ export type TenantCollection = GenericCollection<Tenant>;
 export type AttributeCollection = GenericCollection<AttributeReadmodel>;
 export type PurposeCollection = GenericCollection<PurposeReadModel>;
 export type ClientCollection = GenericCollection<ClientReadModel>;
-export type KeyCollection = GenericCollection<JWKKeyInReadModel>;
+export type KeyCollection = GenericCollection<JWKKey>;
```

Comment:
The types with `ReadModel` suffix are the ones that need adapters for the dates.